doc/data/messages/i/invalid-character-carriage-return/details.rst

```diff
@@ -1 +1,5 @@
-You can help us make the doc better `by contributing <https://github.com/pylint-dev/pylint/issues/5953>`_ !
+This message exists because one of our checker is very generic, but it's never going to
+raise during normal use as it's a ``syntax-error`` that would prevents the python ast
+(and thus pylint) to construct a code representation of the file.
```

Comment:
```suggestion
This message exists because one of our checkers is very generic, but it's never going to
raise during normal use as it's a ``syntax-error`` that would prevent the python ast
(and thus pylint) from constructing a code representation of the file.
```

Reply:
You always find so much mistakes when I write doc ! The ratio of mistake time the fact that I write so much in a given day permits to infer the number of mistakes I make in a day and I'm a little self conscious now 😄 Well at least pylint is free from my bad grammar.